### ISSUE TITLE
Initial Language/Library Decisions

### ISSUE THREAD

**Issue by ghost** [thewanderer41/adventurelookup#1]
From the Discord Server:
We have decided to use for the back-end implementation:
- Python3
- Django
- Docker
- PostGreSQL

**Comment by thewanderer41** [comment]
This is just for the back-end implementation.  Have we decided on using PostgreSQL?

**Comment by ghost** [comment]
Yes I think we also decided that. I have updated the Original Post
